src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionResponseContent.cs

```diff
@@ -16,7 +16,7 @@ private sealed class HttpConnectionResponseContent : HttpContent
             private HttpContentStream _stream;
             private bool _consumedStream;
 
-            public void SetStream(HttpContentReadStream stream)
+            public void SetStream(HttpContentStream stream)
```

Comment:
@baulig I don't know about this one. You hit the WebSocket revert I talked to you about, see e5835204fde so this is only small undo from the big change

Reply:
Was there any specific reason why we did this Mono-specific change?  It pretty much looks like a merging error to me.

Reply:
It was not a merge error but commit revert as the change broke on Mono and we didn't fully investigate at that time